### PR TITLE
OSDOCS#6506: Nutanix - move install config parameters out of assemblies

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.14+

Issue:
This PR addresses [osdocs-6506](https://issues.redhat.com/browse/OSDOCS-6506)

Link to docs preview:
- [Installation configuration parameters for Nutanix](https://64683--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installation-config-parameters-nutanix)
  New assembly at book level.
- Installing a cluster on Nutanix > [Creating the installation configuration file](https://64683--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-nutanix-installer-provisioned#installation-initializing_installing-nutanix-installer-provisioned)
Note the absence of Installation configuration parameters following this topic. In lieu of the module, a cross reference is provided for the new assembly.

It is worth noting that the latter is an example of the change that this PR introduces. This change has been applied to all Alibaba installation assemblies.

QE review:
- [N/A] QE has approved this change.
Given that this update is a reorg only, QE approval is not required. See conversation at https://github.com/openshift/openshift-docs/pull/61165#pullrequestreview-1493117699

### CODE REVIEW

File: installing/installing_nutanix/installation-config-parameters-nutanix.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+[id="installation-config-parameters-nutanix"]
+= Installation configuration parameters for Nutanix
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-nutanix
+:platform: Nutanix
+
+toc::[]
+
+Before you deploy an {product-title} cluster on Nutanix, you provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
+
```
Comment (mjpytlak) [inline installing/installing_nutanix/installation-config-parameters-nutanix.adoc:11]: **Peer review**:
The goal of this PR is to refactor existing content. I acknowledge that this existing content needs some work. Once I have completed the refactor for all providers, I plan on rewriting this and pushing it through QE review. Please see previous reviews

- [Alibaba](https://github.com/openshift/openshift-docs/pull/64049#discussion_r1310262310)
- [IBM Power VS](https://github.com/openshift/openshift-docs/pull/64103#discussion_r1311755915)

**Review by skrthomas — APPROVED** [review]
/lgtm

**Comment by sjstout** [comment]
LGTM

**Pull request merged**